Automattic/newspack-theme
# Add first pass at Spanish translations

### All Submissions:


### Changes proposed in this Pull Request:

This PR does a first pass at adding a Spanish language translation of the theme.

It is all auto-generated using Poedit (the theme largely uses common strings so it filled up quite a bit); it would be great to get it read through by a Spanish-speaker for any corrections and to get the gaps filled in. But for the meantime it should take care of most of the front-facing strings. 

See #278 .

### How to test the changes in this Pull Request:

1. Apply PR.
2. Navigate to WP Admin > Settings > General and change the language to 'Spanish'. 
3. Confirm that most of the theme strings are using Spanish (great places to check are the footer credits, or the top of a single post). 

### Other information:
